code/modules/mob/dead/observer/observer.dm

```diff
@@ -112,6 +175,7 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 
 
 /mob/dead/observer/Move(NewLoc, direct)
+	dir = direct //For the new fancy directional sprites. |- Ricotez
```

Comment:
yeah @duncathan I was going to make that point (I actually did further above) but realised the whole comment is useless so it can all be dropped. but ye.